### PR TITLE
Implement a Jaeger to WF propagator

### PR DESCRIPTION
**Pull request by song-hao**
Address https://github.com/wavefrontHQ/wavefront-opentracing-sdk-python/issues/18

### CODE REVIEW

File: wavefront_opentracing_sdk/span_context.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def get_baggage_item(self, key):
         :return: Baggage value
         :rtype: str
         """
-        return self._baggage[key]
+        return self._baggage[key] if key in self._baggage else None
```
Comment (vikramraman) [inline wavefront_opentracing_sdk/span_context.py]: Wouldn't `return self._baggage[key]` already return None if the key was not present?

Comment (song-hao) [reply wavefront_opentracing_sdk/span_context.py]: It will raise `KeyError` instead of return `None`

Comment (vikramraman) [reply wavefront_opentracing_sdk/span_context.py]: `self._baggage.get(key)` should work. Though I forget if `get` works across 2.x and 3.x

File: .travis.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ python:
 - "3.6"
 - "3.7"
 install:
+- pip install future
```
Comment (ustinov) [inline .travis.yml]: What do we need `future` for?

**Pull request merged**